test/docfx.Test/build/ConfigTest.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+
+namespace Microsoft.Docs.Build
+{
+    public static class ConfigTest
+    {
+        [Theory]
+        [InlineData("locales: [zh-cn]", true, new string[0], new[] { "zh-cn" })]
+        [InlineData("locales: [ zh-cn, de-de ]", true, new string[0], new[] { "zh-cn", "de-de" })]
+        [InlineData("branches:   [master]", true, new[] { "master" }, new string[0])]
+        [InlineData("branches:   [master,   live]", true, new[] { "master", "live" }, new string[0])]
+        [InlineData("branches: [master, live] locales: [zh-cn]", true, new[] { "master", "live" }, new[] { "zh-cn" })]
+        [InlineData("branches: [master, live]     locales: [zh-cn, de-de]", true, new[] { "master", "live" }, new[] { "zh-cn", "de-de" })]
+        [InlineData("branches: [live, live]     locales: [zh-cn, de-de]", true, new[] { "live" }, new[] { "zh-cn", "de-de" })]
+        [InlineData("branches: [LIVE, live]     locales: [zh-cn, ZH-CN]", true, new[] { "LIVE", "live" }, new[] { "zh-cn" })]
+        [InlineData("locales: zh-cn branches: [live]", true, new[] { "live" }, new string[0])]
+        [InlineData("branches: [live] locales: zh-cn", true, new[] { "live" }, new string[0])]
+        [InlineData("branches: live", false, null, null)]
+        [InlineData("locales: zh-cn", false, null, null)]
```

Comment:
If feels incorrect that `branches: [live] locales: zh-cn` passed but `locales: zh-cn` failed

Reply:
should always be array